packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -40,19 +58,22 @@ export function RadioButtonGroup<T>({
     },
     [onChange]
   );
+  const groupName = useRef(uniqueId('radiogroup-'));
```

Comment:
Interesting - why are you using `uniqueId` here?

Reply:
Keyboard controls for a radio group depends on the `name` attribute of all radio inputs being the same, so I didn't want it to be hardcoded which might cause strange behavior if there's more than one `RadioButtonGroup` on one page. To avoid any problems I just decided to generate a unique name for each `RadioButtonGroup`.